whichdata.tex

```diff
@@ -2,11 +2,25 @@ \section{Which data is sensitive ?} \label{sec:which}
 
 In communications thus far and in the security summit held on 6$^{th}$ April 2020 all data has been considered.
 
-We believe the vast majority of the 20TB of nightly images are not of a sensitive nature.
-It would be useful to understand if this is so, especially if delaying this data some period of time is sufficient to desensitize it. If this is the case do we need to encrypt all data ?
+Since then the idea of an Alert Vetting System (AVS) to be implemented by LLNL has been raised.
+A certain set of of potential alerts  would be sent to  and evaluated by AVS.
+These would include  all streaks unattributable to known asteroids that
+\begin{enumerate}
+\item correspond to objects moving faster than vMax=30 deg/day, or
+\item  whose velocity cannot be determined (e.g., due to overlaps with chip boundary).
+\end{enumerate}
+Streaks not forwarded to the AVS would be published as per current baseline.
+Forwarded streaks would be eliminated from Rubin’s public prompt alert stream.
```

Comment:
How does this work? If a streak goes to AVS it will not be part of the stream. If it is ruled out, it will not be part of the stream. How about this: "Forwarded streaks will be vetted. If a streak is determined to be astrophysical by the AVS, it will be included in the prompt product data base and shipped to the Minor Planet Center (MPC) with all other natural streaks. If it is not natural, it will not be returned by the AVS and hence eliminated fro the prompt products database and not sent to the MPC. "

Reply:
I put your text - but the other text is I think what Zeljko and PST had ..